pithos/pithos.py

```diff
@@ -415,6 +433,7 @@ def set_audio_quality(self, *ignore):
         self.worker_run('set_audio_quality', (self.settings.get_string('audio-quality'),))
 
     def pandora_connect(self, *ignore, message="Logging in...", callback=None):
+        self.connected = False
```

Comment:
Trying to setup a signal from the Pandora class. 
```python
    __gtype_name__ = "Pandora"
    __gsignals__ = {
        "connected": (GObject.SignalFlags.RUN_FIRST, None, (GObject.TYPE_BOOLEAN,)),
    }
```
The Pandora class has already has a function named connect so something like self.pandora.connect("connected", self.pandora_signal_handler) in pithos.py won't work for me. I'm sure I'm doing something wrong.